docs/vi/week01/01.md

```diff
@@ -0,0 +1,22 @@
+---
+lang: vi
+lang-ref: 01.md
```

Comment:
What is this? Don't change its original value.

Reply:
I corrected this missing.